### PR TITLE
va-pagination: remove active button background color, override overflow opacity

### PR DESCRIPTION
**Pull request by powellkerry**
## Chromatic

https://2420-pagination-btn-active-color--60f9b557105290003b387cd5.chromatic.com

## Description
Update to sync style for va-pagination with USWDS
- Remove override that made the active page background color blue
- Override overflow opacity to match current USWDS style

USWDS Reference: https://designsystem.digital.gov/components/pagination/
Closes: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2420

## QA Checklist
- [ x] Component maintains 1:1 parity with design mocks
- [ x] Text is consistent with what's been provided in the mocks
- [ x] Component behaves as expected across breakpoints
- [ na - no accessibility changes] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ x] Tab order and focus state work as expected
- [ x] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ x] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ x] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots

Before:
Screenshot 2024-02-16 at 2 04 17 PM

After:
Screenshot 2024-02-16 at 2 03 41 PM

## Acceptance criteria
- [ x] QA checklist has been completed
- [ x] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ x] Documentation has been updated, if applicable
- [ x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by danbrady — APPROVED** [review]
Nice catch on the opacity!

File: packages/web-components/src/components/va-pagination/va-pagination.scss [hunk 1]
```diff
@@ -15,9 +15,8 @@
   a.usa-pagination__previous-page > div {
     display: flex;
   }
-
-  .usa-pagination .usa-current {
-    background-color: var(--color-primary);
+  .usa-pagination__overflow {
+    opacity: inherit;
```
[the remark this answers is not in the document]
Comment (powellkerry) [reply packages/web-components/src/components/va-pagination/va-pagination.scss:19]: That is the conclusion I came to also, but I was a bit worried that if I tried to update our version there might be some issues beyond the scope of this ticket. Should I remove the opacity and write a separate ticket to update USWDS?

[the remark this answers is not in the document]
Comment (jamigibbs) [reply packages/web-components/src/components/va-pagination/va-pagination.scss:19]: @powellkerry Yeah I agree that's out of scope for this. I think it makes sense either way; keep the opacity override here for now and then add a clean-up task to the ticket that does the USWDS update or remove it now. Which ever way you prefer seems fine.

[the remark this answers is not in the document]
Comment (powellkerry) [reply packages/web-components/src/components/va-pagination/va-pagination.scss:19]: Created a new card to update USWDS: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2512

**Pull request merged**